_shared/project/tests/functional/pytest_plugins/cookiecutter.py

```diff
@@ -0,0 +1,24 @@
+{% if cookiecutter.get("postgres") == "yes" %}
+import pytest
+
+from tests.factories.factoryboy_sqlalchemy_session import (
+    clear_factoryboy_sqlalchemy_session,
+    set_factorboy_sqlalchemy_session,
+)
+
+
+@pytest.fixture
+def db(db_engine, db_sessionfactory):
+    """Return a standalone database session for preparing database state."""
+
+    connection = db_engine.connect()
+    session = db_sessionfactory(bind=connection)
+    set_factorboy_sqlalchemy_session(session)
+
+    try:
+        yield session
+    finally:
+        clear_factoryboy_sqlalchemy_session()
+        session.close()
+        connection.close()
+{% endif %}
```

Comment:
I'd like to see if we can get the functests using the same `db` fixture as the unittests do but for now this is copy-pasted from LMS: since it's not obvious how to use the unittests `db` fixture in the functests the functests get this different one